packages/mui-docs/package.json

```diff
@@ -44,6 +44,7 @@
   "devDependencies": {
     "@mui/icons-material": "workspace:*",
     "@mui/material": "workspace:*",
+    "@types/gtag.js": "^0.0.20",
```

Comment:
@mui/code-infra WDYT, would it make sense to specify such types in `dependencies` to avoid the need for explicit definition on the consuming side? 🤔 
https://github.com/mui/mui-x/pull/13999/files#diff-adfa337ce44dc2902621da20152a048dac41878cf3716dfc4cc56d03aa212a56

Reply:
I think it could make sense once `@mui/docs` adopts the `_document.js` implementation (or wherever the GA script is initialized). I'd avoid it to have dependencies declare global variables on dependant projects withoit guaranteeing those will exist.